src/android/aidl/birthday-service/birthday_service/aidl/com/example/birthdayservice/IBirthdayInfoProvider.aidl

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC
+// Copyright 2024 Google LLC
```

Comment:
Nit: `2022-2024` or just keep the original year?

Reply:
This is a new file I created, so I thought we'd want to use 2024 in the comment since that's when it's written?

Reply:
I think using 2024 is correct for new files. My understanding is that we leave the years alone in existing files.

Way back when I first read about licenses for open source projects, the advice was to update the year of files that have received "significant changes" or so. Many projects just updated the year of every file in the beginning of the year. It's now my understanding that we don't do that, we just leave the files alone and only add new notices to new files.